melodic/distribution.yaml

```diff
@@ -4118,18 +4118,18 @@ repositories:
   py_trees:
     doc:
       type: git
-      url: https://github.com/stonier/py_trees.git
-      version: release/0.6-melodic
+      url: https://github.com/splintered-reality/py_trees.git
+      version: release/0.6.x
     release:
       tags:
         release: release/melodic/{package}/{version}
       url: https://github.com/stonier/py_trees-release.git
```

Comment:
Do you want to move the release repository to the new organization too?

Reply:
No, it stays in the same place.